lib/resque/durable.rb

```diff
@@ -66,12 +66,24 @@ def around_perform_manage_audit(*args)
           yield
         end
 
-        a.complete!
+        if requeue_immediately_requested?
+          a.re_enqueue_immediately!
+        else
+          a.complete!
+        end
       else
         yield
       end
     end
 
+    def requeue_immediately_requested?
+      @requeue_immediately
+    end
+
+    def requeue_immediately_requested
```

Comment:
(nit) how about renaming this to `requeue_immediately!`

Reply:
or should it be `re_enqueue_immediately!` to be consistent with the docs

Reply:
I would also change the method `requeue_immediately_requested?` to `requeue_immediately?` for consistency.